src/main/navigation/navigation.c

```diff
@@ -2692,6 +2697,11 @@ static bool rthAltControlStickOverrideCheck(unsigned axis)
                 saveTrackpoint = calculateDistanceToDestination(&posControl.rthTBPointsList[posControl.activeRthTBPointIndex]) > METERS_TO_CENTIMETERS(20);
                 previousTBTripDist = posControl.totalTripDistance;
             }
+
+            // Suspend tracking during loiter on fixed wing. Save trackpoint when loiter enabled.
+            if (distanceCounter && FLIGHT_MODE(NAV_POSHOLD_MODE) && STATE(AIRPLANE)) {
+                saveTrackpoint = suspendTracking = true;
```

Comment:
I find this kind of thing so ugly... you could substitute it for:

saveTrackpoint =  true;
suspendTracking = true

Reply:
Personal choice I guess. I have no problem with it if the types are the same.